lib/airbrake/better_errors.rb

```diff
@@ -0,0 +1,38 @@
+require 'better_errors'
```

Comment:
Please add a comment explaining the purpose of this file. Something like:

"""
BetterErrors module defines middleware that reports uncatched exceptions to Airbrake and displays default error page.
"""

and add whatever details I don't know.